lib/src/Uncertainty/Distribution/ExtremeValueCopula.cxx

```diff
@@ -0,0 +1,339 @@
+  if (conditioningDimension >= getDimension()) throw InvalidArgumentException(HERE) << "Error: cannot compute a conditional CDF with a conditioning point of dimension greater or equal to the distribution dimension.";
+  // Special case for no conditioning or independent copula
+  if ((conditioningDimension == 0) || (hasIndependentCopula())) return x;
+  const Scalar u = y[0];
+  const Scalar v = x;
+  return ConditionalCDF(pickandFunction_, u)(Point(1, x))[0];
+}
+
+/* Compute the quantile of Xi | X1, ..., Xi-1, i.e. x such that CDF(x|y) = q with x = Xi, y = (X1,...,Xi-1) */
+Scalar ExtremeValueCopula::computeConditionalQuantile(const Scalar q, const Point & y) const
+{
+  const UnsignedInteger conditioningDimension = y.getDimension();
+  if (conditioningDimension >= getDimension()) throw InvalidArgumentException(HERE) << "Error: cannot compute a conditional quantile with a conditioning point of dimension greater or equal to the distribution dimension.";
+  if ((q < 0.0) || (q > 1.0)) throw InvalidArgumentException(HERE) << "Error: cannot compute a conditional quantile for a probability level outside of [0, 1]";
+  if (q == 0.0) return 0.0;
+  if (q == 1.0) return 1.0;
+  // Initialize the conditional quantile with the quantile of the i-th marginal distribution
+  // Special case when no contitioning or independent copula
+  if ((conditioningDimension == 0) || hasIndependentCopula()) return q;
+  const Scalar u = y[0];
+  const Scalar res = Brent(SpecFunc::ScalarEpsilon, SpecFunc::ScalarEpsilon, SpecFunc::ScalarEpsilon, 53).solve(ConditionalCDF(pickandFunction_, u), q, SpecFunc::ScalarEpsilon, 1.0 - SpecFunc::ScalarEpsilon);
+  return res;
+}
+
+/* Tell if the distribution has independent copula */
+Bool ExtremeValueCopula::hasIndependentCopula() const
+{
+  return (pickandFunction_(Point(1, 0.5))[0] == 1.0);
+}
+
+/* Pickand function accessor */
+void ExtremeValueCopula::setPickandFunction(const Function & pickandFunction,
+    const Bool check)
+{
+  // Checks on input and output dimensions are done unconditionally
+  if (pickandFunction.getInputDimension() != 1) throw InvalidArgumentException(HERE) << "Error: the Pickand function must have an input dimension equal to 1, here input dimension=" << pickandFunction.getInputDimension();
+  if (pickandFunction.getOutputDimension() != 1) throw InvalidArgumentException(HERE) << "Error: the Pickand function must have an output dimension equal to 1, here input dimension=" << pickandFunction.getOutputDimension();
+  if (check)
+  {
+    // Check if max(t, 1-t) <= A(t) <= 1
+    const UnsignedInteger n = ResourceMap::GetAsUnsignedInteger("ExtremeValueCopula-CheckGridSize");
+    for (UnsignedInteger i = 0; i < n; ++i)
+    {
+      const Scalar t = i / (n - 1.0);
+      const Scalar valueF(pickandFunction(Point(1, t))[0]);
+      const Scalar valueH(pickandFunction.hessian(Point(1, t))(0, 0, 0));
+      if (!((valueF <= 1.0) && (valueF > std::max(t, 1.0 - t))))
```

Comment:
According to the commit log message, should be: `valueF >= std::max(t, 1.0 - t)`